requirements/dev.txt

```diff
@@ -57,7 +57,7 @@ filelock==3.8.0
     #   virtualenv
 idna==3.4
     # via requests
-importlib-metadata==4.12.0
+importlib-metadata==5.0.0
```

Comment:
No breaking change mentioned in the new version release notes. Removed some deprecated entry points. https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v500